celery/backends/couchbase.py

```diff
@@ -103,7 +106,7 @@ def get(self, key):
             return None
 
     def set(self, key, value):
-        self.connection.set(key, value)
+        self.connection.set(key, value, ttl=self.expires)
```

Comment:
A unit test could check that the proper expiration TTL (or `None`) is passed correctly.

Reply:
Thanks for the suggestions! I've updated my PR with unit tests for the expires property (much of it was based off of the redis backend unit tests). Please let me know if there's anything else should change

Reply:
@dfresh613 well done. Just try and fix the Travis CI issues and this should be ready for merge.